### DISCUSSION TITLE
v2.0.0

### DISCUSSION THREAD

**Discussion by imdevarsh** [imdevarsh/meowclient#16] [Announcements]
- Adds forum support
- **BREAKING:** Change responses that return status of request into the full request object! _Please_ remember this!

## What's Changed
* Add forum support by @webdev03 in https://github.com/webdev03/meowclient/pull/15

**Full Changelog**: https://github.com/webdev03/meowclient/compare/v1.1.0...v2.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/webdev03/meowclient/releases/tag/v2.0.0'>v2.0.0</a>.</em>

**Comment by hello-smile6** [comment]
Are you going to register #scratchinfo on IRC? I'm at my channel limit and would like to drop it, but you haven't registered yet.

**Reply by hello-smile6** [reply]
@webdev03

**Reply by imdevarsh** [reply]
Well I don't need an IRC yet, only about 30 users have logged in

**Reply by hello-smile6** [reply]
@webdev03 Can you please just register the channel and install an IRC client so I don't have to manage it?

**Reply by imdevarsh** [reply]
Fine later today

**Reply by hello-smile6** [reply]
> Fine later today

Could you now?
